components/raftstore/src/store/snap.rs

```diff
@@ -742,24 +920,78 @@ impl Snapshot {
     }
 
     fn delete(&self) {
+        macro_rules! try_delete_snapshot_files {
+            ($cf_file: ident, $file_name_func: ident) => {
+                let mut file_id = 0;
+                loop {
+                    let file_path = $cf_file.path.join($cf_file.$file_name_func(file_id));
+                    if file_exists(&file_path) {
+                        delete_file_if_exist(&file_path).unwrap();
+                        file_id += 1;
+                    } else {
+                        break;
```

Comment:
What if the tikv-server instance is killed after some SSTs are deleted? How about keeping try to delete more files?

Reply:
Currently the delete() is on critical path of fsm. Trying to delete more files (such as hard coded 100 files) will significantly add IOs and slow the fsm processing. And even with that, this is still the best effort style. 
Once the snapshot gc is moved to background thread, then we can be more aggressive here. 

Practically the code path should work fine because in most cases, the snapshot files of one CF is only 1.